lib/ansible/plugins/strategy/__init__.py

```diff
@@ -408,7 +410,8 @@ def _queue_task(self, host, task, task_vars, play_context):
 
                     # Pass WorkerProcess its strategy worker number so it can send an identifier along with intra-task requests
                     worker_prc = WorkerProcess(
```

Comment:
Is it possible to change this to the explicit `kwargs` rather than  positional args? I know it's not needed but I personally think having lots of positional args are just hard to read.

Reply:
Sure, I mean it's an internal thing, so we can do whatever we want.  And yeah, that many posargs is a pita.

Reply:
Implemented.